cmake/projects/zookeeper/hunter.cmake

```diff
@@ -0,0 +1,27 @@
+# cmake/projects/Example/hunter.cmake
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cacheable)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    zookeeper
+    VERSION
+    "3.4.9-p1"
```

Comment:
I think it make sense if version and URL matches. I.e. it should be `3.4.9-hunter-7`.